### DISCUSSION TITLE
How to use URock?

### DISCUSSION THREAD

**Discussion by MaryWP** [UMEP-dev/UMEP#506] [Q&A]
Hey,

I would love to test the Wind Model URock in UMEP https://umep-docs.readthedocs.io/en/latest/processor/Wind%20model%20URock.html?highlight=Urock).
However, I can't find it in the processor plugin in QGIS.

Is there anything I need to download or do? Or do I just fail to see the obvious?

Thanks a lot,
Maria

**Comment by j3r3m1** [comment]
Hi Maria,
Thank you for the interest in this module. Sorry for the inconvenience, it is indeed not available yet but will be released in UMEP before April.

**Reply by MaryWP** [reply]
Thanks for the quick reply!
